frontend/src/components/table/TableRowTitleDescription.tsx

```diff
@@ -34,7 +32,7 @@ const TableRowTitleDescription: React.FC<TableRowTitleDescriptionProps> = ({
 
   return (
     <>
-      <b data-testid={testId || `table-row-title-${title}`}>
+      <b data-testid="table-row-title">
```

Comment:
nit: Seems like we could have kept testId as an override for this default value, but if it wasn't being used, I have no issue with this.